src/include/sof/math/sqrt.h

```diff
@@ -0,0 +1,17 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2021 Intel Corporation. All rights reserved.
+ *
+ * Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+ *
+ */
+
+#ifndef __SOF_MATH__SQRTLOOKUP__H
+#define __SOF_MATH__SQRTLOOKUP__H
+
+/* Include Files */
```

Comment:
This comment adds no information. It could be vaguely useful as a section header if there were some big sections but there are none.